### PR TITLE
Update DG

### LINKED ISSUE #164: Update Use Cases

### PR DESCRIPTION
**Pull request by jiaqi20**

### CODE REVIEW

**Review by daongochieu2810 — APPROVED** [review]

**Review by caitlinjee — APPROVED** [review]
lgtm

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -396,14 +402,20 @@ The following sequence diagram shows how delete operation works when `execute(de
 
 ![DeleteSequence](images/implementation/DeleteSequence.png)
 
-<div markdown="span" class="alert alert-info">:information_source: 
-**Note:** 
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Note:**
```
Comment (TianYong-Goh) [inline docs/DeveloperGuide.md]: I think can remove the colon for the note.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -441,14 +453,20 @@ The following sequence diagram shows how edit operation works when `execute("edi
 
 ![EditSequence](images/implementation/EditSequence.png)
 
-<div markdown="span" class="alert alert-info">:information_source: 
-**Note:** 
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Note:**
```
Comment (TianYong-Goh) [inline docs/DeveloperGuide.md]: Same as above

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -492,13 +510,18 @@ Command and Parser make use of Substitutability:
 The following sequence diagram shows how get edit operation works when `execute(editR 1)` or `execute(editF 1)` API call:
 
 ![GetEditSequence](images/implementation/GetEditSequence.png)
-<div markdown="span" class="alert alert-info">:information_source: 
-**Note:** 
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Note:**
```
Comment (TianYong-Goh) [inline docs/DeveloperGuide.md]: same as above

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -560,12 +583,16 @@ The following sequence diagram shows how search operation works when `execute("s
 
 ![SearchSequence](images/implementation/SearchSequence.png)
 
-<div markdown="span" class="alert alert-info">:information_source: 
-**Note:** 
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Note:**
```
Comment (TianYong-Goh) [inline docs/DeveloperGuide.md]: same as above

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -634,12 +661,16 @@ The following sequence diagram shows how clear operation works when `execute(cle
 
 ![ClearSequenceDiagram](images/implementation/ClearSequence.png)
 
-<div markdown="span" class="alert alert-info">:information_source: 
-**Note:** 
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Note:**
```
Comment (TianYong-Goh) [inline docs/DeveloperGuide.md]: same as above

**Review by TianYong-Goh — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jiaqi20) [reply docs/DeveloperGuide.md]: ok removed!

**Review by oliviajohansen — APPROVED** [review]
lgtm

**Pull request merged**